next.config.mjs

```diff
@@ -1,11 +1,11 @@
 import { withSentryConfig } from "@sentry/nextjs";
 import remarkGfm from "remark-gfm";
-import remarkPrism from "remark-prism";
+import rehypePrettyCode from "rehype-pretty-code";
 import createMDX from "@next/mdx";
 import withPWA from "next-pwa";
 import runtimeCaching from "next-pwa/cache.js";
 
-const isProduction = process.env.NODE_ENV === "production";
+//const isProduction = process.env.NODE_ENV === "production";
```

Comment:
Is this no longer needed?

Reply:
It is only referenced by a line in the `pwaConfig` section which is commented out.
I commented it out to get rid of the unused variable squiggly, but left it in case the config line gets uncommented.